code/modules/antagonists/traitor/saboteur/saboteur.dm

```diff
@@ -0,0 +1,17 @@
+/datum/antagonist/traitor/midround/saboteur
+	name = "Syndicate Saboteur"
+	should_equip = FALSE
+	special_role = "saboteur"
+
+/datum/antagonist/traitor/midround/saboteur/greet()
+	..()
+	to_chat(owner.current, "<B><font size=5 color=red>Unlike other syndicate agents, your job is simply to make the lives of the crewmembers harder. Kill only when absolutely necessary.</font></B>")
```

Comment:
From experience, cocky administrators have enforced the anti-murderbone rule on players whose objective is to wreck shit just for killing those handful people who opposed them in a rather messy way in the past.
It'd be more convenient to tell them to stay alive, and don't get caught on their troubles instead. :pensive: